course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -0,0 +1,101 @@
+### The `if` statement
+- the `if` statement is used to check if a conditional is `True`
+- syntax for writing the `if` statement:
+    1. use the `if` keyword at the beginning of the statement
+    2. follow the keyword with a correctly written conditional
+    3. finish the expression with colon 
+    4. block of code that will be executed if the statement passes - should be indented (usually by 4 spaces)
+
+An example of proper writing of the `if` statement:
+
+```py
+friend = "Rolf"
+name = input("Enter your name: ")
+if name == friend:
+    print(f"{name} is a friend")
+```
+
+Two examples of incorrect writing of the `if` statement:
+
+```py
+if name = friend:
+    print(f"{name} is a friend")
+```
+```py
+if name == friend:
+print(f"{name} is a friend")
+```
+**Extra resources**
+- [What is an if statement?](https://python.tecladocode.com/2_countries_visited/2_if_statements.html#what-is-an-if-statement)
+
+### The compound `if` statement - `elif` keyword
+- `elif` keyword is used for chaining multiple if statements
+- `elif` block will be entered only if the previous `if`/`elif` block doesn't get entered
+- the syntax is the same as with writing the `if` statement, the difference being that the `elif` keyword is used instead of `if`
+- `elif` keyword should be indented the same number of times as the previous `if`/`elif` keyword
+
+An example of proper writing of the `if` and `elif` statements:
+
+```py
+best_friend = "Rolf"
+relative = "Phil"
+name = input("Enter your name: ")
+if name == best_friend:
+    print(f"{name} is a best friend")
+elif name == relative:
+    print(f"{name} is a family member")
```

Comment:
I don't know about this example 🤔 . Would the user really know the two names inside the code, in a real wold application?

Reply:
Again, I was thinking of notes as being refreshers on syntax and execution of the code, rather than focusing on other parts of explaining software development. I followed some patterns that Jose uses in his lectures so we can stay consistent with the course but if @jslvtr wants, I can change examples to look more like real-world problems